### DISCUSSION TITLE
Are inputs sanitized by default from the Client Side SDK?

### DISCUSSION THREAD

**Discussion by blakedeckard** [pocketbase/pocketbase#2921] [Q&A] [answered]
Hi, are inputs to the database sanitized by default? Mostly concerned with this from the use of the client side SDK.

I saw this mentioned under file uploading and handling with the sanitized filename but didn't see mentions of other inputs. I checked in the FAQ, all of the basics section, records under web APIs section, database under advanced section, and searched for similar discussions here.

Are there any other known security items that should be considered along with sanitizing inputs?

Thank you!

**Comment by ganigeorgiev** [comment, marked as answer]
All client-side submitted data (including the filter and sort query parameters) should be safe and  don't require extra sanitization when sending requests directly to the PocketBase web APIs.

When using PocketBase as framework, eg. to execute custom queries, then you need to bind user submitted data as db parameters to avoid SQL injections (see [Binding parameters](https://github.com/pocketbase/dbx#binding-parameters); _more detailed documentation and examples on this will be available with the next release_).

**Reply by ganigeorgiev** [reply]
If using the JS SDK, you may want to also check https://github.com/pocketbase/js-sdk#security.

**Reply by blakedeckard** [reply]
Thank you!
